### ISSUE TITLE
Terminal sometimes display nothing (or black spaces) instead of a spaces with a specific background color 

### ISSUE THREAD

**Issue by tigrouind** [microsoft/terminal#15623]
### Windows Terminal version

1.17.11461.0

### Windows build number

10.0.22621.1848

### Steps to reproduce

Run the following C# console program a few times : https://gist.github.com/tigrouind/dc2db97cdd8cde04e3bf6beb21b46509

It print spaces to the console while changing background color, producing a specific pattern (see expected behavior).

From time to time, some write statements are missing and pattern appear broken (see actual behavior). This occurs randomly but it seems to be always when the terminal start to scroll vertically. Also : writing visible characters (eg: letters) instead of spaces seems to prevent bug happening.

### Expected Behavior

![image](https://github.com/microsoft/terminal/assets/2211721/05dcf318-9ba3-498a-9b94-f65cefcca38b)

### Actual Behavior

The pattern appear broken as it contains some black spaces : 

![image](https://github.com/microsoft/terminal/assets/2211721/33353332-0b1f-4d69-a319-1d1d2b66a814)

**Comment by carlos-zamora** [comment]
Looks like this might be fixed by #15618

**Comment by lhecker** [comment]
Ah sorry @carlos-zamora, I didn't mean to imply that #15618 fix this. It's more like that #15618 is in the same category of VtEngine bugs. My hope is that we'll fix this bug some other way sometime soon if I implement passthrough mode and we can slim down VtEngine.

**Comment by zadjii-msft** [comment]
WAIT this looks just like #14512 - Does updating to Terminal Preview, v1.18 fix this/?

**Comment by tigrouind** [comment]
> WAIT this looks just like #14512 - Does updating to Terminal Preview, v1.18 fix this/?

Yes it does. I installed it from Windows Store. The "Atlas Engine" was enabled. Even when disabled, that terminal preview seems to be immune to that rendering glitch.

**Comment by lhecker** [comment]
Thanks for letting us know! I'll close the issue then. 🙂
